packages/analytics/src/index.ts

```diff
@@ -113,8 +110,13 @@ export const createRouteHandler = ({
           throw new Error("Could not fetch IP address");
         }
 
-        const { country } = await maxMindClient.country(ip);
-        countryISOCode = country?.isoCode ?? "Unknown";
+        const geoData = await fetch(`http://ip-api.com/json/${ip}`).then((res) => res.json());
+
+        if(geoData.status === "success") {
+          countryISOCode = geoData.countryCode;
+        } else {
+          throw new Error(geoData.message);
```

Comment:
I think we probably want to specifically check for the 429 response code so we can log something else if we are throttled.